### ISSUE TITLE
Move cgroupv1 specific code out of libcontainers/cgroups pkg

### ISSUE THREAD

**Issue by kolyshkin** [opencontainers/runc#2471]
The following public identifiers should be moved from `github.com/opencontainers/runc/libcontainer/cgroups` to a separate package (say `github.com/opencontainers/runc/libcontainer/cgroups/cgroupv1`). The reason is, they should **only** be used from cgroupv1 code (i.e. when `cgroups.IsCgroup2UnifiedMode()` returns `false`), and they return errors otherwise.

* CgroupNamePrefix
* IsNotFound
* FindCgroupMountpoint
* FindCgroupMountpointAndRoot
* Mount
* GetOwnCgroup
* GetCgroupMounts
* GetOwnCgroup
* GetOwnCgroupPath
* GetInitCgroup
* GetInitCgroupPath

In addition, these two should be made private:

* NotFoundError
* NewNotFoundError

**Comment by kolyshkin** [comment]
Originally tacked in https://github.com/opencontainers/runc/pull/2411 but the package renaming commit removed for now.

**Comment by danishprakash** [comment]
@kolyshkin I'd like to work on this if it's something that external contributors can pick up. As you mentioned, the changes you made in `libcontainers/cgroups/utils.go` still require a package rename, does it make sense for me to start from there?

**Comment by danishprakash** [comment]
Makes sense, that being said, would you mind If I ask you to point me to an issue that a new contributor can work on? (bg: been using runc in production for over 2 years now and would really like to contribute back). Thanks.
